lua/legendary/extensions/nvim_tree.lua

```diff
@@ -0,0 +1,25 @@
+local function init()
```

Comment:
We're waiting until the user has called setup, so user mappings will be present.